packages/sanity/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@next-auth/sanity-adapter",
+  "description": "Sanity Adapter for NextAuth",
+  "version": "1.2.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "author": "Fedeya <elfedeminaya@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "keywords": [
+    "next-auth",
+    "oauth",
+    "sanity",
+    "next.js"
+  ],
+  "scripts": {
+    "setup": "yarn build",
+    "build": "tsc"
+  },
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "devDependencies": {
+    "@types/lru-cache": "^5.1.0"
+  },
+  "dependencies": {
+    "@sanity/client": "^2.8.0",
+    "argon2": "^0.27.2",
+    "axios": "^0.21.1",
```

Comment:
```suggestion
```
I think we wouldn't need `axios`, as `fetch` will be available, thanks to `Next.js`